vanguard.py

```diff
@@ -0,0 +1,49 @@
+import sys
+from pymarc import Record, Field, MARCReader, MARCWriter
+
+#SILSLA-13
+
+def modify_5xx(record):
+	for old_field in range(500, 600):
+		if old_field != 590:
+			for fld in record.get_fields(old_field):
+				for sfld in fld.get_subfields('5'):
+					if sfld.startswith('CLU'):
+						fld.tag = '590'
+						record.add_ordered_field(fld)
+	return record
+
+def change_CLU(record):
```

Comment:
I think this function should be renamed.  It does the same basic thing as `modify_5xx()`, so should have a similar name.
Since there is a lot of duplicated code, maybe use the `field_mapping` approach: add a helper function which adds the 99 relevant entries from the range (500-599, minus 590) to `field_mapping`, then you could use `change_CLU()` and remove `modify_5xx()`.